scripts/apothecary/formulas/FreeImage/FreeImage.sh

```diff
@@ -10,20 +10,20 @@
 FORMULA_TYPES=( "osx" "vs" "win_cb" "ios" "android" )
 
 # define the version
-VER=3160 # 3.16.0
+VER=3170 # 3.16.0
```

Comment:
Are these updates required?  We already updated a bunch of the static libs and I'm a little worried about not having matching for all platforms.

Also, the comment 3.16.0 should be fixed if we stay with this.